wave_lang/kernel/wave/gather_to_shared.py

```diff
@@ -307,10 +325,10 @@ def gather_to_shared(
         THREAD_2: THREAD_2 if waves_per_block[2] > 1 else 0,
     }
 
-    supported_load_widths = [32]
+    supported_load_widths = [4]
```

Comment:
I kind of prefer the old 32, 96, 128 since it is more analogous to the instructions, any thoughts on doing the division inside `get_load_bitwidh` or maybe we can omit them altogether since we can change `vector_width = (elements_per_thread * bitwidth) // 8` to `vector_width = (elements_per_thread * bitwidth)`. Thoughts?

Reply:
In this case we will need to convert bounds to bitwidth, which is also weird.